README.md

```diff
@@ -1,18 +1,37 @@
 # rg_instructor_analytics
 
+## Installation
+* Add `rg_instructor_analytics` to the installed app 
```

Comment:
You write the readme file for all potential users of the application not only DevOps engineers. So please add the paths in all places which need to be updated.

Reply:
done